src/Setup/WixBA/RootView.xaml

```diff
@@ -1,125 +1,487 @@
 <?xml version="1.0" encoding="utf-8" ?>
-<!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the Microsoft Reciprocal License. See LICENSE.TXT file in the project root for full license information. -->
-
+<!--  Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the Microsoft Reciprocal License. See LICENSE.TXT file in the project root for full license information.  -->
 
 <Window x:Class="WixToolset.UX.RootView"
-    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-    xmlns:ux="clr-namespace:WixToolset.UX"
-    WindowStartupLocation="CenterScreen"
-    WindowStyle="None"
-    AllowsTransparency="True"
-    Background="{x:Null}"
-    Width="400"
-    Height="400">
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:i="http://schemas.microsoft.com/expression/2010/interactivity"
+        xmlns:ux="clr-namespace:WixToolset.UX"
+        MinWidth="550"
+        MinHeight="400"
+        MaxWidth="1200"
```

Comment:
The WebBrowser control does have a MaxWidth value sit which is just
slightly smaller that the Window/@MaxWidth.  That is one of the few times
that I limit the width of a control but in this case when I mocked an
available update and added lots of 'content' HTML if made since to allow
the dialog to be almost the width of a sheet of paper (on my monitors).
But to allow the main window to continue to expand just looks strange.
Also as I re-listen to the 109 meeting, while demo-ing the Appx install Rob
noted that they also had set a Window MinWidth and I don't know if they had
a MaxWidth.  I will go and remove the MaxWidth (along with the Mouse
handling which is a different issue, and based on my testing that will
degrade the UX somewhat, but I am sure it will be acceptable to most).

On Mon, Aug 1, 2016 at 11:45 AM, Sean Hall <notifications@github.com> wrote:

> In src/Setup/WixBA/RootView.xaml
> <https://github.com/wixtoolset/wix4/pull/208#discussion_r73011188>:
>
> > -    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
> > -    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
> > -    xmlns:ux="clr-namespace:WixToolset.UX"
> > -    WindowStartupLocation="CenterScreen"
> > -    WindowStyle="None"
> > -    AllowsTransparency="True"
> > -    Background="{x:Null}"
> > -    Width="400"
> > -    Height="400">
> > +        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
> > +        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
> > +        xmlns:i="http://schemas.microsoft.com/expression/2010/interactivity"
> > +        xmlns:ux="clr-namespace:WixToolset.UX"
> > +        MinWidth="550"
> > +        MinHeight="400"
> > +        MaxWidth="1200"
>
> If the WebBrowser control is the issue, then let's put a max size on that
> instead of the window (or just make the WebBrowser a fixed size?).
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/wixtoolset/wix4/pull/208/files/28721ab8e4985a5641b3efd7975a67cecd8f8d4a#r73011188>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHyCLQVsH5F1llGzqig1K8ZFVqnT96JTks5qbiKqgaJpZM4JVm8P>
> .
>